app/helpers/accounts_helper.rb

```diff
@@ -17,6 +17,10 @@ def payment_source_link_or_text(account)
     end
   end
 
+  def show_account_facilities_tab?(ability, account)
```

Comment:
I’m pretty sure `current_ability` is a view helper, so I think it would be available inside this method. I don’t think it needs to be passed in as an argument.

Reply:
I know, but I’m not a fan of relying on shared state in helper methods (because that state is outside of the module), so I try to make them take all the arguments they need; are you okay with it like this, or do you think consistency (and using `current_ability`) is preferable?

Reply:
I’m good with it. Your logic is sound. :shipit: (damn. The custom emoji don’t work on the mobile version)

Reply:
Oh, they do! They just don’t auto suggest

Reply:
lol 😂